myosuite/envs/myo/myochallenge/chasetag_v0.py

```diff
@@ -156,7 +156,7 @@ def chase_player(self):
         new_vec = np.array([np.cos(theta), np.sin(theta)])
         new_vec2 = pel - vec
         vel = np.dot(new_vec, new_vec2)
-        return np.array([1.0, vel])
+        return np.array([self.rng.uniform(1, 5), vel])
```

Comment:
@elladyr is this wanted here?

Reply:
I think we need to set vel per reset, not sample it everytime in this function

Reply:
Sample chaser velocities corrected.